File: ui-integration/src/main/java/io/snabble/sdk/ui/integration/ProductSearchFragment.java
package io.snabble.sdk.ui.integration;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.fragment.app.Fragment;

public class ProductSearchFragment extends Fragment {
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.snabble_fragment_productsearch, container, false);
    }
}

File: ui/src/main/java/io/snabble/sdk/ui/search/ProductSearchFragment.kt
package io.snabble.sdk.ui.search

import android.os.Bundle
import android.view.View
import io.snabble.sdk.ui.BaseFragment
import io.snabble.sdk.ui.R

open class ProductSearchFragment : BaseFragment(R.layout.snabble_fragment_productsearch) {
    override fun onActualViewCreated(view: View, savedInstanceState: Bundle?) {
        val v = view as ProductSearchView
        v.allowAnyCode = true
    }
}